File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/attivita/dettagli/Trappola.ts
import { Ditta } from "app/Model/metaschema/Ditta";
import {BaseCodeDescr} from "../../baseClass/baseCodeDescr";


export class Trappola extends BaseCodeDescr {

    DurataFeromone: number;
    Ditta: Ditta;
    Scadenza: Date;

    constructor(codice: number) {
        super(codice);
    }
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/attivita/dettagli/Trappola.cs
using AgronicaCoreModelsSTD.baseClass;
using AgronicaCoreModelsSTD.metaschema;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.attivita.dettagli
{
    public class Trappola : BaseCodeDescr
    {
        public int DurataFeromone { get; set; }
        public Ditta Ditta { get; set; }
        public DateTime Scadenza { get; set; }

        public Trappola(int codice) : base(codice, "")
        {

        }

        public Trappola() : base(-1, "")
        {
        }
    }
}
